### PR TITLE
Add Termux Installation Docs

### PR DESCRIPTION
**Pull request by lordbagel42**
I have no idea what page you want this set to, please change it at the top of the file.

### CODE REVIEW

File: server-setup/termux-installation [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: [[no idea what to set this to]]
```
Comment (NWBx01) [inline server-setup/termux-installation]: nav_order is the order in which sections appear on the sidebar. If you don't want to move the other sections, just set it to 4.

File: tools/termux-installation.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: [[no idea what to set this to]]
```
Comment (Louka3000) [inline tools/termux-installation.md:3]:
```suggestion
nav_order: 2
```

That would make it appear right below owotracker.
You'll need to modify lime-slimes.html to 3 and shadowpc.html to 4 tho

File: server-setup/osc-information.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+parent: SlimeVR setup
+---
+
+# OSC
+
+OSC stands for Open Sound Control. It was originally developed to be used to connect and control music equipment in a simple, open, and standard way. VRChat developed a system for it to be used to control the avatar and virtual trackers, allowing for fully body tracking on Quest 2. You can read the VRChat documentation on it [here](https://docs.vrchat.com/docs/osc-overview).
+
+# OSC Setup
+
+Once you have setup your trackers and mounted them correctly, you can setup OSC. In the SlimeVR Gui, navigate to Settings>OSC>VRChat. If the server is running on the Quest, you can leave it as 127.0.0.1. If the server is not on the Quest, put in your Quest's IP. You can get this by opening the Quick Settings menu on your Quest, selecting Wifi, your currently connected network, and scroll down and click the arrow. The IP address is listed there.
```
Comment (NWBx01) [inline server-setup/osc-information.md:13]:
```suggestion
Once you have setup your trackers and mounted them correctly, you can setup OSC. In the SlimeVR Server, navigate to Settings>OSC>VRChat. If the server is [running on the Quest](../tools/termux-installation.html), you can leave it as 127.0.0.1. If the server is not on the Quest, put in your Quest's IP. You can get this by opening the Quick Settings menu on your Quest, selecting Wifi, your currently connected network, and scroll down and click the arrow. The IP address is listed there.

```

File: server-setup/osc-information.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+## Trackers to enable depending on amount:
+
+- 5+0: waist and feet.
+- 5+1: *chest*, waist and feet.
+- 5+3: *chest*, waist, *knees*, feet.
+- 7+3: *chest*, waist, *knees*, feet, *elbows*.
```
Comment (NWBx01) [inline server-setup/osc-information.md:20]:
```suggestion

```

This section is redundant and repeated from other pages within same section.

File: server-setup/osc-information.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+Now that you have the SlimeVR side setup, you can now connect VRChat to SlimeVR.
+### PRESS RESET IN THE GUI NOW
```
Comment (NWBx01) [inline server-setup/osc-information.md:23]:
```suggestion
### PRESS RESET IN THE SERVER NOW
```

Lots of uses of "GUI" instead of just saying "server".

Comment (lordbagel42) [reply server-setup/osc-information.md:23]: There's a difference between the server and GUI, thought it made more sense.

File: tools/termux-installation.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+Using this, you can install Termux, a linux "emulator" for Android. The Quest 2 happens to run android, so you can host the SlimeVR server on it using this.
+
+***joycons still require a PC for now and the Wrangler: https://github.com/carl-anders/slimevr-wrangler
+
+# INSTRUCTIONS - QUEST 2
+1. Install a file explorer on your quest, CX File Explorer is good. You can get the APK from https://apkpure.com
+
+2. Download Termux (quest 1/2 is arm64) from here: https://github.com/termux/termux-app/releases
+    DO NOT INSTALL FROM GOOGLE PLAY.
```
Comment (ButterscotchV) [inline tools/termux-installation.md:17]: I don't think you even can do this on a Quest? Either delete this or make this consistent with the other line :P
```suggestion
```
```suggestion
    **DO NOT INSTALL FROM GOOGLE PLAY.**
```

File: tools/termux-installation.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+nav_order: 2
+parent: Applications and Variants
+---
+
+# What is this?
+
+Using this, you can install Termux, a linux "emulator" for Android. The Quest 2 happens to run android, so you can host the SlimeVR server on it using this.
```
Comment (ButterscotchV) [inline tools/termux-installation.md:9]: This paragraph is a little awkward.
```suggestion
This guide will help you install Termux, a Linux "emulator" for Android to use for SlimeVR. The Quest 2 happens to run Android, so you can run the SlimeVR server on it using Termux.
```

File: tools/termux-installation.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+***joycons still require a PC for now and the Wrangler: https://github.com/carl-anders/slimevr-wrangler
+
+# INSTRUCTIONS - QUEST 2
+1. Install a file explorer on your quest, CX File Explorer is good. You can get the APK from https://apkpure.com
```
Comment (ButterscotchV) [inline tools/termux-installation.md:14]: Do you need to? The other instructions don't seem like they would.

File: tools/termux-installation.md [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+```
+5. Go to https://beta-slimevr-gui.bscotch.ca/ on the device with the server.
+
+# Remote GUI Access
+
+The first step is enabling this on whatever browser you use: https://www.damirscorner.com/blog/posts/20210528-AllowingInsecureWebsocketConnections.html
+  Enable it for https://beta-slimevr-gui.bscotch.ca/
+Now that you have done this, you can go to https://beta-slimevr-gui.bscotch.ca/?ip=[QUEST OR PHONE IP}&port=21110 and access the site. Substitute [QUEST OR PHONE IP] with your quest or phones IP address, no brackets.
+
+Credit goes to Butterscotch!#2066 for making all of this work.
```
Comment (ButterscotchV) [inline tools/termux-installation.md:51]: ;3
```suggestion
Credit goes to Butterscotch!#2066 for making all of this work and Bagel#6705 for the documentation, idea, and motivation.
```

File: server-setup/osc-information.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+Now that you have the SlimeVR side setup, you can now connect VRChat to SlimeVR.
+### PRESS RESET IN THE SERVER NOW
+
+Open the [VRChat - Beta](https://www.oculus.com/experiences/quest/4821132827998004/) on your Quest.
+
+Using the Radial Menu, select Options at the top, OSC, and then enable it.
+
+![img](https://user-images.githubusercontent.com/737888/154179201-ec413948-7013-494a-81fb-4b5e1129cf5f.jpg)
+
+Walk over to your mirror, open the non-radial menu, and press Calibrate FBT. Make a T-Pose, lining yourself up with the T-Pose in the mirror. Pull both triggers, and you've connected VRChat to SlimeVR with OSC.
```
Comment (ButterscotchV) [inline server-setup/osc-information.md:25]:
```suggestion
Walk over to your mirror, open the non-radial menu, and press Calibrate FBT. Make a T-Pose, lining yourself up with the T-Pose in the mirror. Pull both triggers, and you've connected VRChat to SlimeVR with OSC.

Created by Bagel#6705. SlimeVR OSC implementation by Erimel#7159.
```

**Review by ButterscotchV — CHANGES_REQUESTED** [review]

File: tools/termux-installation.md (hunk 8 above)
Comment (lordbagel42) [reply tools/termux-installation.md:14]: It made sense for the original way the installer worked, forgot to change it.

**Review by ButterscotchV — APPROVED** [review]

**Review by calliePepper — APPROVED** [review]
Nice work

**Pull request merged**